back-end/datastore.js

```diff
@@ -0,0 +1,80 @@
+require('dotenv').config({ path: '../.env'});
+
+const { pick } = require('lodash');
+const pg = require('pg-promise')();
+const db = pg({
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  database: process.env.DB_NAME,
+  user: process.env.DB_USER
+});
+
+let sequelize = (obj) => {
```

Comment:
This looks awesome. For now, could you add in a few comments so I can get a better understanding of the data flow? It looks great, though. Nice use of a function factory!